tensorflow/src/TensorFlow/Nodes.hs

```diff
@@ -96,6 +99,21 @@ instance Nodes ControlNode where
 instance a ~ () => Fetchable ControlNode a where
     getFetch _ = return $ pure ()
 
+-- hm, annoying but doable.
+-- unfortunately, pulling out requires a "Identity" everywhere...bleh
```

Comment:
Added (|:|) which is the version with Identity.

I feel it should be possible to simplify things, but haven't cracked it yet (and it may require ghc-8).  If I figure it out I'll send another PR.